host/4/bullseye/amd64/java/java17/java17-composite.template

```diff
@@ -0,0 +1,38 @@
+ARG JAVA_VERSION=17.0.3
+ARG JAVA_HOME=/usr/lib/jvm/msft-17-x64
+FROM mcr.microsoft.com/dotnet/runtime-deps:6.0
+ARG HOST_VERSION
+ARG JAVA_VERSION
+ARG JAVA_HOME
+
+COPY --from=runtime-image [ "/FuncExtensionBundles", "/FuncExtensionBundles" ]
+COPY sshd_config /etc/ssh/
+COPY start.sh /azure-functions-host/
+COPY --from=runtime-image [ "/azure-functions-host", "/azure-functions-host" ]
+
+COPY --from=runtime-image [ "/workers/java", "/azure-functions-host/workers/java" ]
+
+EXPOSE 2222 80
+
+ENV AzureWebJobsScriptRoot=/home/site/wwwroot \
+    HOME=/home \
+    FUNCTIONS_WORKER_RUNTIME=java \
+    DOTNET_USE_POLLING_FILE_WATCHER=true \
+    HOST_VERSION=${HOST_VERSION} \
+    ASPNETCORE_CONTENTROOT=/azure-functions-host \
+    JAVA_HOME=${JAVA_HOME}
+
+# Fix from https://github.com/GoogleCloudPlatform/google-cloud-dotnet-powerpack/issues/22#issuecomment-729895157
+RUN apt-get update && \
+    apt-get install -y libc-dev wget libfreetype6 fontconfig fonts-dejavu && \
```

Comment:
Not important for this PR but, calling out that this change may no longer be necessary.  We are apt-get ing libc-dev as a result of a GRPC issue from 2020.  Reference : https://github.com/GoogleCloudPlatform/google-cloud-dotnet-powerpack/issues/22#issuecomment-729895157

Reply:
Thank you very much for the heads up. Will validate and remove if it's unnecessary in a separate pr.